packages/admin-ui/src/filters/filter-popover.tsx

```diff
@@ -12,11 +12,17 @@ import { FilterStatus } from './filter-status'
 
 export const FilterPopover = createComponent<typeof Menu, FilterPopoverOptions>(
   (props) => {
-    const { children, state, ...restProps } = props
+    const { children, state, onRetry = () => {}, ...restProps } = props
 
     const [isScrollableLayout, setIsScrollableLayout] = useState(false)
     const contextState = { isScrollableLayout, setIsScrollableLayout, state }
 
+    const shouldDisplayChildren = !(
+      state.status === 'error' ||
```

Comment:
**Suggestion**: Probably, it would be nice to have all those status as a enum, i think it would help to know the possible filter status to be used. What do you think? 🤔